### ISSUE TITLE
Is it possible to parse json like editly to create video

### ISSUE THREAD

**Issue by desis123** [mifi/reactive-video#19]
I am just wondering is it possible to parse editly like json to create video with reactive-video ?

**Comment by mifi** [comment]
no, although someone could probably make an adapter for it. note that transitions will not work because I don't know if it's possible to implement transitions in html
